scripts/md2rst.py

```diff
@@ -29,5 +29,5 @@ def download_pandoc():
 def md2rst(text):
     download_pandoc()
     output = pypandoc.convert_text(text, 'rst', format='md',
-     filters=[os.path.join('filters', 'yml_filter.py')])
+     filters=[os.path.join(os.getcwd(),'filters', 'yml_filter.py')])
```

Comment:
This is incorrect. this only works if the current working directory is the scripts directory which it clearly isn't. Please use the correct path and test with travis properly.